miner/bid_simulator.go

```diff
@@ -128,8 +128,8 @@ func newBidSimulator(
 ) *bidSimulator {
 	// Set default value
 	maxBids := uint32(3)
-	if config.MaxBidsPerBuilder > 0 {
-		maxBids = config.MaxBidsPerBuilder
```

Comment:
before this PR, config.MaxBidsPerBuilder may not set, here config.MaxBidsPerBuilder  we get is 0,
so code 130~133, will set maxBids to `3`

after this PR, config.MaxBidsPerBuilder is always set, configured or make it default.
so code 129~133 can be delete,
and let code 144
maxBidsPerBuilder :  *config.MaxBidsPerBuilder